osquery/events/linux/auditdnetlink.cpp

```diff
@@ -668,6 +579,155 @@ bool AuditdNetlink::configureAuditService() noexcept {
   return true;
 }
 
+bool AuditdNetlink::clearAuditConfiguration() noexcept {
+  int seq = audit_request_rules_list_data(audit_netlink_handle_);
+  if (seq <= 0) {
+    VLOG(1) << "Failed to list the audit rules";
+    return false;
+  }
+
+  // Attempt to list all rules
+  std::vector<AuditRuleDataObject> rule_object_list;
+  auto timeout = getUnixTime() + 5;
+
+  for (size_t read_retry = 0U; read_retry < 3U; ++read_retry) {
+    if (timeout < getUnixTime()) {
+      VLOG(1) << "Failed to unconfigure the audit service (timeout)";
+      return false;
+    }
+
+    bool netlink_ready = false;
+
+    for (size_t poll_retry = 0U; poll_retry < 3U; ++poll_retry) {
+      pollfd fds[] = {{audit_netlink_handle_, POLLIN, 0}};
+
+      errno = 0;
+      int poll_status = ::poll(fds, 1, 4);
+      if (poll_status == 0) {
+        continue;
+      }
+
+      if (poll_status < 0) {
+        VLOG(1) << "poll() failed with errno " << errno;
```

Comment:
nit, most of your other `VLOG` lines have an `Audit:` prefix

Reply:
I don't have a preference one way or the other (include the `Audit:` prefix, or don't) since the log lines already include the filename and line number, but let's remain consistent. Keep in mind that no other part of the code base includes a prefix like this.

Reply:
You are right, I have removed the prefixes to make it more consistent with the rest of the code!